blog/2020-05-08-AIVersusCovid-19.adoc

```diff
@@ -0,0 +1,135 @@
+= AI versus Covid-19: How Java helps nurses and doctors in this fight
+:page-interpolate: true
+:awestruct-author: Christopher-Chianelli
+:awestruct-layout: blogPostBase
+:awestruct-tags: [use case]
```

Comment:
Add a share_image_filename line like in the other blog posts. It goes into the the html head as a meta info so when social media show that url, it show that picture along.
So that share_image should be something that attracks, for example a screenshot or so.